packages/salesforcedx-vscode-core/src/conflict/persistentStorageService.ts

```diff
@@ -70,6 +72,8 @@ export class PersistentStorageService {
   }
 
   public makeKey(type: string, fullName: string): string {
-    return `${type}#${fullName}`;
+    const orgUserName = workspaceContext.username;
+    const projectPath = getRootWorkspacePath();
+    return `${orgUserName}#${projectPath}#${type}#${fullName}`;
```

Comment:
This looks good. Just for a sanity check, can we test if memento has any limitation on the length of the key?

Reply:
Great question! I tested with key lengths of 0.1k chars, 1k chars, and 10k chars. 0.1k chars and 1k chars work as expected. 10k chars would return a `SIGPIPE` error, but I think the chance is really slim that keys exceed 1k chars.